schemas/private/ekscluster-kfd-v1alpha2.json

```
@@ -584,6 +587,33 @@
         "publicAccess"
       ]
     },
+    "Spec.Kubernetes.NodePoolsCommon": {
+      "type": "object",
+      "additionalProperties": false,
+      "description": "All the common self-managed node pool definitions. Currently supports only the IMDS properties.",
```

what do you think of:
```suggestion
      "description": "Additional properties in common for all self-managed node pools. Currently only IMDS properties are supported.",
```